src/passes/stringify-walker.h

```diff
@@ -65,6 +65,97 @@ struct StringifyWalker
 
   using Super = PostWalker<SubType, UnifiedExpressionVisitor<SubType>>;
 
+  struct SeparatorCtx {
+    struct FuncStart {
+      Function* func;
+    };
+
+    struct FuncEnd {
+      Function* func;
+    };
```

Comment:
Can this just be `End` as well, or do we need the function pointer?

Reply:
done